matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/alias/AddRoomAliasTask.kt

```diff
@@ -16,28 +16,34 @@
 
 package org.matrix.android.sdk.internal.session.room.alias
 
+import org.greenrobot.eventbus.EventBus
+import org.matrix.android.sdk.internal.di.UserId
 import org.matrix.android.sdk.internal.network.executeRequest
-import org.matrix.android.sdk.internal.session.room.RoomAPI
+import org.matrix.android.sdk.internal.session.directory.DirectoryAPI
+import org.matrix.android.sdk.internal.session.room.alias.RoomAliasAvailabilityChecker.Companion.toFullAlias
 import org.matrix.android.sdk.internal.task.Task
-import org.greenrobot.eventbus.EventBus
 import javax.inject.Inject
 
 internal interface AddRoomAliasTask : Task<AddRoomAliasTask.Params, Unit> {
     data class Params(
             val roomId: String,
-            val roomAlias: String
+            val aliasLocalPart: String
```

Comment:
Do we need to add a comment on top of this? Or should we expect developers to understand what is the local part of an alias?

Reply:
I tried to use `aliasLocalPart` and `alias` when appropriate to distinguish the case, since there are both String. But yeah I will add a comment here.